.github/workflows/release.yml

```diff
@@ -221,6 +221,18 @@ jobs:
     - name: Create or update version branch      
       run: git push origin $(git rev-parse HEAD):refs/heads/$BRANCH_VERSION
 
+    - name: Configure Operator SDK
+      if: ${{ needs.initialize.outputs.release_type == 'minor' }}
+      run: |
+        export ARCH=$(case $(uname -m) in x86_64) echo -n amd64 ;; aarch64) echo -n arm64 ;; *) echo -n $(uname -m) ;; esac)
+        export OS=$(uname | awk '{print tolower($0)}')
+
+        export OPERATOR_SDK_DL_URL=https://github.com/operator-framework/operator-sdk/releases/download/v1.20.0
```

Comment:
this probably isn't good grabbing the hardcoded 1.20. We use very specific version of the SDK. Our Makefile has a target you can use that downloads the SDK we use: https://github.com/kiali/kiali-operator/blob/master/Makefile#L63

Reply:
Thanks for the recommendation. It's resolved now.